ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -306,6 +306,18 @@ def test_human_output(self, restart_service):
         assert output == \
             "ERROR: ipahealthcheck.meta.services.sssd: sssd: not running"
 
+    def test_human_output(self):
+        """
+        Test if in case no  failures were found, informative string is printed
+        in human output.
+
+        https://pagure.io/freeipa/issue/8892
+        """
+        returncode, output = run_healthcheck(self.master, output_type="human",
+                                             failures_only=True)
+        assert returncode == 0
+        assert output == "No issues found.\n"
```

Comment:
run_healthcheck strips the output so drop the trailing \n and I think this will pass.